webapp/sources/rudder/rudder-web/src/main/elm/rules/sources/DataTypes.elm

```diff
@@ -230,10 +230,13 @@ type alias RuleDetails = { originRule : Maybe Rule, rule : Rule, tab :  TabMenu,
 
 type alias CategoryDetails = { originCategory : Maybe (Category Rule), category : Category Rule, parentId : String, tab :  TabMenu}
 
+type alias RuleNodesDirectives = { id: String, numberOfNodes: Int, numberOfDirectives: Int }
```

Comment:
i'm really not sure of that, rather than upgrading the RuleDetails with these info